README.md

```diff
@@ -1,13 +1,20 @@
-# gcloud SDK package for NPM
-This package is an [NPM][npm] wrapper around the [gcloud SDK][gcloud-sdk] installer for [Google Cloud Platform][gcp]. It supports OSX, Windows, and Linux.
+# Google Cloud SDK package for npm
+This package is an [npm][npm] wrapper around the [Cloud SDK][cloud-sdk] installer for [Google Cloud Platform][gcp]. It supports OSX, Windows, and Linux.
 
-If the gcloud SDK is not present on your machine, this package will install it. Otherwise, it will update the existing installation using `gcloud components update`.
+If the Cloud SDK is not present on your machine, this package will install it. Otherwise, it will update the existing installation using `gcloud components update`.
 
 ## Use
-```
+
+You can install the Cloud SDK for use globally:
+```sh
 npm install -g @google-cloud/cloud-sdk
 ```
 
+Or use it as a dev dependency:
+```sh
+npm install --save-dev @google-cloud/cloud-sdk
```

Comment:
Does this install it globally? Currently, it has caveats if installed non-globally that aren't idiomatic to Node.js developers (e.g. settings are not isolated between projects).